src/Agent.Worker/TaskRunner.cs

```diff
@@ -571,6 +578,8 @@ private void PrintTaskMetaData(Definition taskDefinition)
 
         private void PublishTelemetry(Definition taskDefinition, HandlerData handlerData)
         {
+            if (IsCustomerTask()) return;
```

Comment:
nipick: `return` statement should be on the new line in {} brackets
ref: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#layout-conventions